doc/classes/@GlobalScope.xml

```diff
@@ -29,9 +29,6 @@
 		<member name="Geometry" type="Geometry" setter="" getter="">
 			[Geometry] singleton
 		</member>
-		<member name="GodotSharp" type="GodotSharp" setter="" getter="">
-			[GodotSharp] singleton
```

Comment:
Hm..

Reply:
Not sure how those happened, I just ran doctool a few times.

Reply:
Only the mono build outputs this GodotSharp singleton. For the other issue, not sure.